core/modules/storyviews/classic.js

```diff
@@ -33,7 +33,7 @@ ClassicStoryView.prototype.navigateTo = function(historyInfo) {
 	this.listWidget.dispatchEvent({type: "tm-scroll", target: targetElement});
 };
 
-ClassicStoryView.prototype.insert = function(widget) {
+ClassicStoryView.prototype.insert = function(widget,bottom) {
```

Comment:
Rather than passing this "bottom" flag, perhaps we can update the insert method to take a second parameter called "options", and pass members for the index of the item, and the total number of items?

Reply:
changed it, @Jermolene